pkg/controller/container-runtime-config/helpers.go

```diff
@@ -954,3 +1006,67 @@ func generateSigstoreRegistriesdConfig(clusterScopePolicies map[string]signature
 	}
 	return data, nil
 }
+
+func addScopeMirrorsSigstoreRegistriesdConfig(registriesDockerConfig map[string]dockerConfig, scope string, reg *sysregistriesv2.Registry, sigstoreAttachment dockerConfig) error {
+	if reg == nil {
+		return nil
+	}
+
+	dummyDigest := "@sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+	dummyTag := ":aaa"
+	dummyPath := "/a"
+	dummyPrefix := "matched"
+
+	if strings.HasPrefix(scope, "*.") {
+		scope = strings.Replace(scope, "*", dummyPrefix, 1)
+	}
+
+	scopeRef, err := reference.Parse(scope)
+	if err != nil {
+		return fmt.Errorf("error parsing scope %s: %w", scope, err)
+	}
+
+	if digestRef, digested := scopeRef.(reference.Digested); digested {
+		namedRef, err := reference.ParseNamed(digestRef.String())
+		if err != nil {
+			return fmt.Errorf("error parsing scope digested reference %s: %w", digestRef.String(), err)
+		}
+		scope = reference.TrimNamed(namedRef).String()
+
+	}
+	if tagRef, tagged := scopeRef.(reference.Tagged); tagged {
+		namedRef, err := reference.ParseNamed(tagRef.String())
+		if err != nil {
+			return fmt.Errorf("error parsing scope tagged reference %s: %w", tagRef.String(), err)
+		}
+		scope = reference.TrimNamed(namedRef).String()
+	}
+	if !strings.Contains(scope, "/") {
+		scope += dummyPath
```

Comment:
Ugh… I didn’t realize there’s this corner case.

It’s safe to do because we have already looked up `reg`, but, AFAICS, if we add `dummyPath` here, we must also strip it from the returned endpoints.